doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -585,12 +577,11 @@ bash ./get_source.sh
 When you have all the source files that you need, run the configure script, which detects how to build in the current build environment.
 
 ```
-bash configure --with-freemarker-jar=/<my_home_dir>/freemarker.jar \
-               --with-boot-jdk=<path_to_macOS_JDK10> \
+bash configure --with-boot-jdk=<path_to_macOS_JDK10> \
```

Comment:
This should be updated to advise using jdk11 for the boot jdk like other platforms.

Reply:
I'll open a PR soon to fix a number of these issues, just need to port the changes to jdk14.
Here is a preview https://github.com/eclipse/openj9/compare/master...pshipton:docprereq?expand=1

Reply:
https://github.com/eclipse/openj9/pull/9487